basyx/aas/adapter/xml/xml_deserialization.py

```diff
@@ -85,8 +85,9 @@ def _tag_replace_namespace(tag: str, nsmap: Dict[str, str]) -> str:
     """
     split = tag.split("}")
     for prefix, namespace in nsmap.items():
-        if namespace == split[0][1:]:
-            return prefix + ":" + split[1]
+        if prefix:
```

Comment:
Which value does `prefix` have when the prefix is missing? I would prefer an explicit check for that value (e.g. `prefix is not None`, if `prefix` were to be `None` when the prefix is missing).